native/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ["Andre Staltz <andre@staltz.com>"]
 license = "AGPL-3.0"
 build = "build.rs"
 exclude = ["artifacts.json", "index.node"]
+edition = "2018"
```

Comment:
What's the `edition`? Not familiar to me

Reply:
It's a way to opt in to some breaking language changes. "2018" is basically a language version bump. Modules using different editions are still compatible.

https://doc.rust-lang.org/edition-guide/rust-2018/edition-changes.html

Reply:
Not mentioned on the page is that `extern crate` isn't required (in most cases) in 2018.

https://doc.rust-lang.org/reference/items/extern-crates.html